util/Setup/EnvironmentFileBuilder.cs

```diff
@@ -70,12 +70,12 @@ private void Init()
                 ConnectTimeout = 30,
                 TrustServerCertificate = true,
                 PersistSecurityInfo = false
-            }.ConnectionString;
+            }.ConnectionString.Replace("\"", "\\\"");
```

Comment:
This change is confusing to me. Why would someone have double quotes in their connection string itself? It doesn't seem like `SqlConnectionStringBuilder` would produce a connection string that contains them.

Reply:
After further testing it seems my previous code change of enclosing the SQL connection string in single quotes works fine.  I had an error in how I was testing images.